Exercises.Delegate/Exercises.Delegate/Point.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+
+namespace Exercises.Delegate
+{
+    public class Point : IComparable
+    {
+
+        public enum PointRepresentation { Polar, Rectangular }
+
+        private double r, a;
```

Comment:
1. New declaration on new line for each variable;
2. What is `r`, `a`? Please use more meaningful names;
3. Private variables are started with `_`. Example: `_point`
4. Add `readonly` modifier if variable is not expected to be modified. (Not here, but just note for future)
5. You can also consider using autoproperties for `r` and `a`